tests/functional/test_generatecliskeleton.py

```diff
@@ -12,7 +12,7 @@
 # language governing permissions and limitations under the License.
 import json
 
-from ruamel import yaml
```

Comment:
We should probably be able to remove this import all together. It does not look used.